src/main/java/be/syntra/devshop/DevshopFront/models/dto/ProductDto.java

```diff
@@ -14,8 +18,16 @@
 public class ProductDto {
 
     private Long id;
+
+    @NotBlank(message = "I need a name")
     private String name;
+
+    @NotNull(message = "Madam or Sir, I have value!")
```

Comment:
don't try to be cute. Rather say: This is a required field. Goes for all messages. Don't use "I need...", but something like -> "positive value required".